src/text/raw_text_writer.rs

```diff
@@ -9,15 +9,134 @@ use crate::raw_symbol_token_ref::{AsRawSymbolTokenRef, RawSymbolTokenRef};
 use crate::result::{illegal_operation, IonResult};
 use crate::types::decimal::Decimal;
 use crate::types::timestamp::{Precision, Timestamp};
+use crate::types::ContainerType;
 use crate::writer::Writer;
 use crate::{Integer, IonType};
 
+pub struct RawTextWriterBuilder {
+    space_between_values: String,
+    indentation: String,
+    space_after_field_name: String,
+    space_after_container_start: String,
+}
```

Comment:
🗺️ There are two big changes in this file. First, I've added a builder API that allows whitespace in the raw text writer to be customized. Second, I've used this functionality to allows applications to make a 'pretty' writer.